src/main/java/seedu/address/logic/commands/ListEventCommand.java

```diff
@@ -31,9 +32,29 @@ public ListEventCommand(Comparator<Event> comparator) {
         this.comparator = comparator;
     }
 
+    public boolean isOverdue(Event event) {
+        LocalDateTime currentDateTime = LocalDateTime.now();
+        LocalDateTime eventDateTime = event.getEventDate().getDateTime();
+        if (currentDateTime.isAfter(eventDateTime)) {
+            return true;
+        }
+        return false;
+    }
+
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
+
+        /*
+        int length = model.getEventsList().size();
+        for (int i = 0; i < length; i++) {
+            Event event = model.getEventsList().get(i);
+            if (isOverdue(event)) {
+                model.deleteEvent(event);
+            }
+        }
+        */
```

Comment:
Delete if not using O:

Reply:
Let me try to fix first, if it doesn't work by Friday I'll delete hahaha